content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy from the foo namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample applications sleep and httpbin:
```

Comment:
Can we rephrase this to: Delete sample applications `sleep` and `httpbin` from the `foo` namespace: (Put the names in backpacks ``)